crates/l2/contracts/bin/system_contracts_updater/main.rs

```diff
@@ -7,15 +7,15 @@ use error::SystemContractsUpdaterError;
 use ethrex_common::U256;
 use ethrex_common::types::GenesisAccount;
 use ethrex_l2::utils::test_data_io::read_genesis_file;
-use ethrex_l2_sdk::{COMMON_BRIDGE_L2_ADDRESS, L1_MESSENGER_ADDRESS, compile_contract};
+use ethrex_l2_sdk::{COMMON_BRIDGE_L2_ADDRESS, L2_TO_L1_MESSENGER_ADDRESS, compile_contract};
 use genesis_tool::genesis::write_genesis_as_json;
 mod cli;
 mod error;
 
 fn main() -> Result<(), SystemContractsUpdaterError> {
     let opts = SystemContractsUpdaterOptions::parse();
     compile_contract(&opts.contracts_path, "src/l2/CommonBridgeL2.sol", true)?;
-    compile_contract(&opts.contracts_path, "src/l2/L1Messenger.sol", true)?;
+    compile_contract(&opts.contracts_path, "src/l2/L2ToL1Messenger.sol", true)?;
```

Comment:
Can this contract change in the future? Consider this scenario:
1. You update the code, this yields a new bytecode.
2. You then add the bytecode to the genesis file like what happens below, on lines 42-45.

If the contract was already deployed, this would result in a different genesis file from the original one, right?

Of course, feel free to correct me.